### ISSUE TITLE
`--stdout` always trigger an error (ErrCode 3088)

### ISSUE THREAD

**Issue by vincentbernat** [dagwieers/unoconv#156]
While `unoconv -f pdf -o out.pdf input.doc` works without a problem, `unoconv -f pdf --stdout input.doc > /dev/null` gives this error:

```
unoconv: UnoException during export phase:
Unable to store document to private:stream (ErrCode 3088)

Properties: ((com.sun.star.beans.PropertyValue){ Name = (string)"Overwrite", Handle = (long)0x0, Value = (any){ (boolean)true }, State = (com.sun.star.beans.PropertyState)DIRECT_VALUE }, (com.sun.star.beans.PropertyValue){ Name = (string)"FilterName", Handle = (long)0x0, Value = (any){ (string)"writer_pdf_Export" }, State = (com.sun.star.beans.PropertyState)DIRECT_VALUE }, (com.sun.star.beans.PropertyValue){ Name = (string)"OutputStream", Handle = (long)0x0, Value = (any){ (com.sun.star.uno.XInterface)0x2142228{, supportedInterfaces={com.sun.star.lang.XTypeProvider,com.sun.star.io.XOutputStream}} }, State = (com.sun.star.beans.PropertyState)DIRECT_VALUE })
```

This is with LibreOffice 4.1.0.

This also has been reported to Debian as [bug #718481](http://bugs.debian.org/718481)

**Comment by Merlyn007** [comment]
I have a problem like this

I need to convert a bulk of .doc files but i have problems.

unoconv -vvv -f jpg *.doc

Properties: ((com.sun.star.beans.PropertyValue){ Name = (string)"OutputStream", Handle = (long)0x0, Value = (any){ (com.sun.star.uno.XInterface)0x7fdc0cf5ce28{, supportedInterfaces={com.sun.star.io.XOutputStream,com.sun.star.lang.XTypeProvider}} }, State = (com.sun.star.beans.PropertyState)DIRECT_VALUE }, (com.sun.star.beans.PropertyValue){ Name = (string)"Overwrite", Handle = (long)0x0, Value = (any){ (boolean)true }, State = (com.sun.star.beans.PropertyState)DIRECT_VALUE }, (com.sun.star.beans.PropertyValue){ Name = (string)"FilterName", Handle = (long)0x0, Value = (any){ (string)"draw_jpg_Export" }, State = (com.sun.star.beans.PropertyState)DIRECT_VALUE })
Input file: M10804.doc
Selected output format: Joint Photographic Experts Group [.jpg]
Selected office filter: draw_jpg_Export
Used doctype: graphics
Output file: M10804.jpg
unoconv: UnoException during export phase:
Unable to store document to file:///Users/macbook-lucas/Desktop/tapas/M10804.jpg (ErrCode 3088)

I am running under OSX YOSEMITE
with libreoffice 4.3.2

Thanks for the help.

**Comment by dagwieers** [comment]
We have changed a few things in the master branch and I was wondering if you all see this issue with the current master branch. If so, shout here !

**Comment by vincentbernat** [comment]
I am unable to reproduce the problem with LibreOffice 4.4.4 (even with 0.6).

**Comment by dagwieers** [comment]
Just as a reference, ErrCode 3088 means:

Binary: 000:01100:00010000

    Area: 0 (ERRCODE_AREA_TOOLS)
    Class: 12 (ERRCODE_CLASS_WRITE)
    Code: 16 (ERRCODE_IO_CANTWRITE)

So that seems to indicate a permission/ACL/SELinux issue ?

**Comment by vincentbernat** [comment]
That would be odd for stdout. Maybe in the past, LO was trying to seek to be able to write to stdout?

**Comment by dagwieers** [comment]
Hmmm, in light of the fix in #170 it is clear that not just permission/access problems are a cause for ErrCode 3088, but also other inabilities to write (like incorrect code).

I have been adding the ErrCode to the subject of issues in the hope of finding a common theme. And I have been documenting how it is  translated here: https://github.com/dagwieers/unoconv/blob/master/doc/errcode.adoc

**Comment by regebro** [comment]
Seems like nobody is having this issue any more. Closing.
